apps/web/src/pages/get-started/components/get-started-tabs/GetStartedTabs.tsx

```diff
@@ -33,6 +35,13 @@ export const GetStartedTabs: React.FC<IGetStartedTabsProps> = ({ tabConfigs = TA
         mb={15}
       >
         <Tabs.List>
+          <Tabs.Tab
+            key={`tab-${OnboardingUseCasesTabsEnum.ECHO}`}
+            value={OnboardingUseCasesTabsEnum.ECHO}
+            icon={<IconConnectedTv style={ICON_STYLE} />}
+          >
+            Echo Studio
```

Comment:
I think we should remove Studio from here. Studio is a mere detail here. The main point is Code-First ala Echo only. Pretty sure we have adjusted this on Figma. Is that the latest copy?

Reply:
We did but Radek liked to have the studio there and now the latest is Workflows will push it when done with other changes.